kernel/data-pipeline/core/src/main/java/org/apache/shardingsphere/data/pipeline/common/sqlbuilder/PipelineImportSQLBuilder.java

```diff
@@ -44,12 +46,20 @@ public final class PipelineImportSQLBuilder {
     
     private final PipelineSQLSegmentBuilder sqlSegmentBuilder;
     
-    private final ConcurrentMap<String, String> sqlCacheMap;
+    private final Cache<String, String> sqlCache;
     
-    public PipelineImportSQLBuilder(final DatabaseType databaseType) {
+    public PipelineImportSQLBuilder(final DatabaseType databaseType,final ShardingCacheOptionsConfiguration cacheOptions) {
         dialectSQLBuilder = DatabaseTypedSPILoader.getService(DialectPipelineSQLBuilder.class, databaseType);
         sqlSegmentBuilder = new PipelineSQLSegmentBuilder(databaseType);
-        sqlCacheMap = new ConcurrentHashMap<>();
+        sqlCache = buildRouteCache(cacheOptions);
+    }
+
+    private Cache<String, String> buildRouteCache(final ShardingCacheOptionsConfiguration cacheOptions) {
```

Comment:
1, `buildRouteCache` could be renamed, since there's no route here.

2, From `new ShardingCacheOptionsConfiguration(true, 1, 1)`, maximum size could not be `1`.

Reply:
1. yes,i need to rename,thanks
2.I refer to other code.maybe, i should  modify to 100 and 1024

Reply:
That's OK